source/documentation/support/support.md

```diff
@@ -0,0 +1,34 @@
+## Troubleshooting 
+
+### Error codes
+
+| Error code | Description           |
+|------------|-----------------------|
+| 200        | Ok                    |
+| 404        | Data not found        |
+| 500        | Internal server error |
+
+### Finding a specific record
+
+If you cannot find a specific record, check you have the correct field value. You must use the exact field value to get a match from the register. For example, in the `local-authority-eng` register, you must capitalise the field value: 
+
+Incorrect request: https://local-authority-eng.register.gov.uk/record/bir
+
+Correct request: https://local-authority-eng.register.gov.uk/record/BIR
+
+### Adding additional data 
+
+If you want to combine register data with data from other sources, you may find it useful to keep the data from each source in separate files and combine them at a later stage. For example, you could keep separate files for data about countries and for data about territories.
```

Comment:
I'm not following what do you mean here.

Reply:
This is a holdover from before my time. It seems faintly useless to me but I assumed it was important.

Reply:
@arnau I'll open a separate PR to address this since this one's approved.